lib/plottr_components/src/components/dashboard/files/FilesHome.js

```diff
@@ -37,6 +45,21 @@ const FilesHomeConnector = (connector) => {
     }
   }
 
+  function scrivenerImportDialog() {
+    const title = t('Choose your Scrivener project')
+    const filters = [{ name: t('Scrivener file'), extensions: ['scriv'] }]
+    const properties = ['openFile', 'createDirectory']
```

Comment:
These options don't work for me.  I need `'openDirectory'` in the list for it to work on Linux, but when I add it, I have to remove the `extensions` filter on the above line.
@cameronsutter can you comment here?
What have you used to open folders in the past (Scrivener files are a bit of an edge case).